app/istqb-ct-ste-complete-guide.css

```
@@ -0,0 +1,469 @@
+.arch-desc { font-size: 1rem; color: var(--color-text-secondary); }
+
+/* ===== PYRAMID ===== */
+.pyramid { display: flex; flex-direction: column; align-items: center; gap: 0.5rem; margin: 1.5rem 0; }
+.pyramid-level {
+  display: flex; align-items: center; justify-content: center;
+  border-radius: var(--radius-DEFAULT, 12px); padding: 0.75rem 1.5rem;
+  font-family: var(--font-mono); font-size: 1rem; font-weight: 700;
+  transition: all 0.25s cubic-bezier(0.4, 0, 0.2, 1); cursor: default;
+  text-align: center;
+}
+.pyramid-level:hover { transform: scaleX(1.04); box-shadow: 0 0 20px currentColor; }
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)